auth0_flutter/analysis_options.yaml

```diff
@@ -32,7 +32,6 @@ linter:
     test_types_in_equals: true
     throw_in_finally: true
     unnecessary_statements: true
-    unsafe_html: true
```

Comment:
`unsafe_html` is removed in Dart 3.7. The project had no ignores, so this can be removed.